_posts/2023-01-31-xbuddies.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: 'X-Buddies - Code Camp Crossword Fun'
+author: 'detlef'
+featuredImage: x-buddies
+excerpt: "Why should crossword enthusiasts always solve a puzzle on their own? Wouldn't it be much more fun and enjoyable to solve a crossword online with others? Let us introduce the Code Camp 2022 project 'X-Buddies'."
+permalink: '/2023/01/31/x-buddies.html'
+categories: [Code Camp, React, JavaScript, SSE, Docker]
```

Comment:
[Code Camp, React, Typescript, Spring Boot, Java, SSE, Docker]

Reply:
Will change the categories and merge it into master.